src/style.css

```diff
@@ -0,0 +1,88 @@
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+}
+
+.container {
+  margin: 20px 0;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+}
+
+.container:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.input {
+  width: 100%;
+  color: #3d4e61;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  font-family: Avenir, sans-serif;
+}
+
+input:focus {
+  font-family: Avenir, sans-serif;
+  font-weight: 800;
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+}
+
+.loop {
+  position: absolute;
+  z-index: 1;
+  top: 25px;
+  left: 26px;
+}
+
+.loop-small {
+  width: 11px;
+  height: 11px;
+  position: absolute;
+  z-index: 1;
+  top: 15px;
+  left: 13px;
+}
+
+.big {
+  position: relative;
+}
+
+.big-input {
+  display: block;
+  padding: 25px 62px;
+  font-size: 16px;
+  line-height: 22px;
+  height: 70px;
+}
+
+.small {
+  position: relative;
+}
+
+.small-input {
+  display: block;
+  padding-top: 12px;
+  padding-left: 33px;
+  padding-bottom: 10px;
+  height: 42px;
+  font-size: 14px;
+}
```

Comment:
point 2 of checklist  https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md

Reply:
there should be one common selector (`.input`) and 1 for small: `input--small`

Reply:
the same for containers, etc